Makefile

```diff
@@ -87,19 +87,31 @@ docker-build-local:
 helm:
 	$(MAKE) -C modules helm
 
-.PHONY: docker-retag-images
-docker-retag-images:
-	docker tag ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/manager:${DOCKER_TAGNAME} ghcr.io/the-mesh-for-data/manager:${DOCKER_TAGNAME}
-	docker tag ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/secret-provider:${DOCKER_TAGNAME} ghcr.io/the-mesh-for-data/secret-provider:${DOCKER_TAGNAME}
-	docker tag ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/egr-connector:${DOCKER_TAGNAME} ghcr.io/the-mesh-for-data/egr-connector:${DOCKER_TAGNAME}
-	docker tag ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/movement-controller:${DOCKER_TAGNAME} ghcr.io/the-mesh-for-data/movement-controller:${DOCKER_TAGNAME}
-	docker tag ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/dummy-mover:${DOCKER_TAGNAME} ghcr.io/the-mesh-for-data/dummy-mover:${DOCKER_TAGNAME}
-	docker tag ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/opa-connector:${DOCKER_TAGNAME} ghcr.io/the-mesh-for-data/opa-connector:${DOCKER_TAGNAME}
-	docker tag ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/vault-connector:${DOCKER_TAGNAME} ghcr.io/the-mesh-for-data/vault-connector:${DOCKER_TAGNAME}
-
-.PHONY: docker-push-public
-docker-push-public:
-	DOCKER_HOSTNAME=ghcr.io DOCKER_NAMESPACE=the-mesh-for-data DOCKER_TAGNAME=${DOCKER_TAGNAME} $(MAKE) docker-push
+DOCKER_PUBLIC_HOSTNAME ?= ghcr.io
+DOCKER_PUBLIC_NAMESPACE ?= the-mesh-for-data
+DOCKER_PUBLIC_NAMES := \
+	manager \
+	secret-provider \
+	egr-connector \
+	movement-controller \
+	dummy-mover \
+	opa-connector \
+	vault-connector
+GITHUB_TAG := $(shell git describe --exact-match --tags HEAD 2>/dev/null)
```

Comment:
Quick comment, Travis has environment variable for it already

Reply:
ok, changed to TRAVIS_TAG